art-community-platform/backend/app/api/views/user.py

```diff
@@ -68,10 +68,20 @@ def get_user_by_id(req, user_id):
     if user_id in request_sender.followings:
         is_following = True
 
+    followers = get_followers_by_user_id_helper(user_id=user_id)
+    followings = get_followings_by_user_id_helper(user_id=user_id)
+    art_item_list = get_art_items_by_user_id_helper(user_id=user_id)
```

Comment:
Is there a need for this 3 lines?

Reply:
They are used in lines 75,76,77. Sorry!

Reply:
Yes, I call for helper functions to get follower/following/art-item list. After that I get their counts.